cadquery/occ_impl/geom.py

```diff
@@ -852,3 +858,72 @@ def isInside(self, b2: "BoundBox") -> bool:
             return True
         else:
             return False
+
+
+class Location(object):
+    """Relative location in 3D space"""
+
+    wrapped: TopLoc_Location
+
+    @overload
+    def __init__(self) -> None:
+        ...
+
+    @overload
+    def __init__(self, t: Vector) -> None:
+        ...
+
+    @overload
+    def __init__(self, t: Plane) -> None:
+        ...
+
+    @overload
+    def __init__(self, t: Plane, v: Vector) -> None:
+        ...
+
+    @overload
+    def __init__(self, t: TopLoc_Location) -> None:
+        ...
+
+    @overload
+    def __init__(self, t: Vector, ax: Vector, angle: float) -> None:
+        ...
+
+    def __init__(self, *args):
```

Comment:
This chunk of code might benefit from a comment or two.

Reply:
Added description for every overload.